src/common/utils/backup.js

```diff
@@ -57,3 +65,112 @@ export function ensureBackupFullPath() {
 
   ensureBackupTodayPath()
 }
+
+export function deleteOldBackups(strategy, amount) {
+  const files = sortFileNamesByDate(backupFiles(BACKUP_BASE_PATH))
+
+  switch (strategy) {
+    case 'days': {
+      const anchorDate = nDaysAgo(amount)
+      const filesToDelete = files.filter((file) => !fileIsSoonerThan(anchorDate, file))
+      console.warn('`Removing old backups: ${filesToRemove}')
```

Comment:
let's make this a `log.warn` from `electron-log` so it shows up in the computer's log (and error reports)